### DISCUSSION TITLE
Apakah akan diadakan fitur pengecekan asset menggunakan barcode?

### DISCUSSION THREAD

**Discussion by sukabumikode** [ruswan/app-management#28] [Q&A] [answered]
Jika diadakan sepertinya akan lebih baik

**Comment by ruswan** [comment, marked as answer]
Sepertinya itu sangat memungkinkan, namun sebelum itu harus dibuatkan dulu fitur cetak barcodenya, soalnya belum ada

**Comment by sukabumikode** [comment]
Baik, terimakasih
